release/release_data_tests.yaml

```diff
@@ -289,17 +289,19 @@
       format: [numpy, pandas, pyarrow]
       compute: [tasks]
       scaling: [fixed_size]
+      map-batches-sleep-ms: 0
     adjustments:
-      # Include at least one test with actors.
+      # Fixed-size actor test.
       - with:
           format: numpy
           compute: actors
           scaling: fixed_size
-      # Include at least one autoscaling test.
+      # Autoscaling tests
       - with:
           format: numpy
-          compute: tasks
+          compute: [actors, tasks]
```

Comment:
I think you'd need to split this up in to two separate `- with` adjustments, one for tasks and one for actors. I don't think you can use a list of items with `with`